app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -24,13 +24,14 @@
             android:id="@+id/feeds_filtered_message"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:paddingStart="60dp"
+            android:paddingHorizontal="16dp"
             android:paddingEnd="0dp"
```

Comment:
Please remove this, it is covered by the horizontal padding

Reply:
paddingStart has already been removed

Reply:
But not paddingEnd

Reply:
Ok, I get it now.